src/net_processing.cpp

```diff
@@ -4076,31 +4076,15 @@ class CompareInvMempoolOrder
 };
 }
 
-bool PeerManager::SendMessages(CNode* pto)
+void PeerManager::MaybeSendPing(CNode& pto)
```

Comment:
If you're changing every reference to `pto` anyway because it's now a reference not a pointer, maybe also call it `node_to` or similar, since it's not a "p" anymore?

Reply:
Sure. I'll split the commit into a move and a rename/change to reference to make it easier to review.

Reply:
Done